cms/static/js/views/pages/container.js

```diff
@@ -40,11 +40,19 @@ define(['jquery', 'underscore', 'backbone', 'gettext', 'js/views/pages/base_page
             initialize: function(options) {
                 BasePage.prototype.initialize.call(this, options);
                 this.viewClass = options.viewClass || this.defaultViewClass;
+                this.isUnitPage = this.options.isUnitPage;
+                this.isSplitTest = (this.model.attributes.category === 'split_test');
```

Comment:
If the only place we don't want it is the library page we can just check if the category is 'library' and render it for every other category